Makefile

```diff
@@ -258,7 +258,7 @@ ci-verify:
 # This is used for release builds by .github/workflows/build.yml
 build:
 	@echo "--> Building Vault $(VAULT_VERSION)"
-	@go build -v -tags "$(GO_TAGS)" -ldflags " -X github.com/hashicorp/vault/sdk/version.Version=$(VAULT_VERSION) -X github.com/hashicorp/vault/sdk/version.GitCommit=$(VAULT_REVISION) -X github.com/hashicorp/vault/sdk/version.BuildDate=$(VAULT_BUILD_DATE)" -o dist/
+	@go build -v -tags "$(GO_TAGS)" -ldflags " -s -w -X github.com/hashicorp/vault/sdk/version.Version=$(VAULT_VERSION) -X github.com/hashicorp/vault/sdk/version.GitCommit=$(VAULT_REVISION) -X github.com/hashicorp/vault/sdk/version.BuildDate=$(VAULT_BUILD_DATE)" -o dist/
```

Comment:
I think technically the `-w` is redundant, as `-s` is a superset. In my testing both `-s` and `-s -w` produced byte-for-byte identical results on a different package, but no harm in keeping it. [[ref]](https://pkg.go.dev/cmd/link)

> -s
>	Omit the symbol table and debug information.

> -w
>	Omit the DWARF symbol table.

Reply:
I noticed this in my testing as well. All of the recommendations I've found say to use both, so I did.